__tests__/__fixtures__/relative-ref-oas/petstore.json

```diff
@@ -106,6 +106,30 @@
           }
         ]
       }
+    },
+    "/user": {
```

Comment:
Added this because I was using this spec to test tag reduction and I needed more than just the `pet` tag in here to make sure we didn't get back `user` from the reducer.